src/analyses/threadEscape.ml

```diff
@@ -32,7 +62,21 @@ struct
 
   (* transfer functions *)
   let assign ctx (lval:lval) (rval:exp) : D.t =
-    ctx.local
+    let ask = Analyses.ask_of_ctx ctx in
+    let lvs = mpt ask (AddrOf lval) in
+    ignore (Pretty.printf "assign lvs %a: %a\n" CilType.Location.pretty !Tracing.current_loc D.pretty lvs);
```

Comment:
Those printouts should be conditional, they produce quite a bit of output.

Reply:
I changed them to tracing.